src/App.js

```diff
@@ -251,78 +258,85 @@ function App() {
 
 
   // //For "blurring" out the passwords for login/sign up pages:
-  const handleShowPassLogin = () => {
-    setLoginValues({ ...loginValues, showPass: !loginValues.showPass })
-  }
-
-  const handleShowPassSignup = () => {
-    setSignupFormValues({ ...signupFormValues, showPass: !signupFormValues.showPass })
-  }
-
-  const handleShowConfirmPassSignup = () => {
-    setSignupFormValues({ ...signupFormValues, showConfirm: !signupFormValues.showConfirm })
+  const handleShowPass = (id) => {
+    if (window.location.pathname === `/login`) {
```

Comment:
This could be simplified a little bit / made more readable 

` if (window.location.pathname === `/login`) {
      setLoginValues({ ...loginValues, showPass: !loginValues.showPass })
    }
  if(window.location.pathname === `/signup`){
    if( id === 'input-pass-icon'){
      ...
    } 
    if(id === 'input-confirm-pass-icon'){
     ... 
    }
  }`